### PR TITLE
TGC-217 happy path

### PR DESCRIPTION
**Pull request by Andrew-Folga**
Adult cattle housing grant happy path changes.

### CODE REVIEW

**Review by alanplatt — APPROVED** [review]

**Review by alanplatt — APPROVED** [review]

File: app/helpers/contants.js
Comment (ScenarioFactory) [inline app/helpers/contants.js:1]: Typo in new file name?

Comment (Andrew-Folga) [reply app/helpers/contants.js:1]: Good spot - pushed now.

**Review by ScenarioFactory — APPROVED** [review]

**Review by alanplatt — APPROVED** [review]

File: app/helpers/handlers.js [hunk 1]
```diff
@@ -48,86 +47,50 @@ const getReplacementText = (request, key, questionType, questionKey, trueReturn,
...
-
-
-  if (url === 'current-multi-tier-system') {
-    const replacement =  getYarValue(request, 'poultryType') === getQuestionAnswer('poultry-type', 'poultry-type-A1', ALL_QUESTIONS) ? 'an' : 'a'
-    field = field.replace('[[_article_]]', replacement)
-  }
-
-  return field
+const insertYarValue = (field, request) => {
+  return field.replace(SELECT_VARIABLE_TO_REPLACE, (_ignore, additionalYarKeyName) => field.includes('£') ? formatUKCurrency(getYarValue(request, additionalYarKeyName) || 0) : getYarValue(request, additionalYarKeyName))
```
Comment (Daveawb) [inline app/helpers/handlers.js:51]: This line is a bit hard to read, especially with the duplication. Would recommend setting the the result of 'getYarValue(request, additinalYarKeyName)' to a variable first

Comment (DavidBarker-EE) [reply app/helpers/handlers.js:51]: Thinking about it all of this functionality should probably belong with `formatUKCurrency` in common.

Comment (Andrew-Folga) [reply app/helpers/handlers.js:51]: We can discuss that but just extracting `getYarValue(request, additionalYarKeyName)` won't work as `additionalYarKeyName` is only known within the `replace` function.

**Pull request merged**